api/handlers_test.go

```diff
@@ -53,9 +60,9 @@ func TestCreateSessionHandlerFunc(t *testing.T) {
 
 			Convey("Then the expected success response is returned", func() {
 				So(resp.Code, ShouldEqual, 201)
-				So(resp.Header().Get("Content-Location"), ShouldEqual, fmt.Sprintf("/session/1234"))
 				So(mockSessions.NewCalls(), ShouldHaveLength, 1)
 				So(mockSessions.NewCalls()[0].Email, ShouldEqual, "test@test.com")
+				So(resp.Body.String(), ShouldEqual, fmt.Sprintf(`{"id":"1234","email":"test@test.com","start":"%s"}`, currentTime.Format(dateTimeFMT)))
```

Comment:
Nothing wrong with this but formatting a json string is a bit messy and will become uglier if the session object becomes more complex

```
...

expected := &session.Session{
	ID:    "1234",
	Email: "test@test.com",
	Start: currentTime,
}

...

b, err := json.Marshal(expected)
So(err, ShouldBeNil)
So(resp.Body.String(), ShouldEqual, string(b))
```
Is a bit easier to read and is easier to maintain.